src/App.ts

```diff
@@ -860,22 +862,23 @@ export default class App {
             // Don't process the last item in the listenerMiddleware array - it shouldn't get a next fn
             const listener = listenerMiddleware.pop();
 
-            if (listener !== undefined) {
-              return processMiddleware(
-                listenerMiddleware,
-                listenerArgs as AnyMiddlewareArgs,
+            if (listener === undefined) {
```

Comment:
In this part, we have to explicitly, instead of implicitly, return undefined in the case the listener is undefined. I like to nest/indent the exceptional case and return right away, which allows me to do further processing below this conditional nested one indent less. So that's what I did here. 🤷

Reply:
I think this is fine.